src/main/webapp/auth.js

```diff
@@ -35,12 +34,32 @@ function init() {
           gapi.auth2.init({
             'client_id': clientId,
           }).then(() => {
+            handleAuthenticationState();
             renderButton();
           });
         });
   });
 }
 
+/**
+ * Will initialize the page based on the current authentication state
+ */
+function handleAuthenticationState() {
+  const featureContainer = document.querySelector('.feature-container');
+  const logInButton = document.querySelector('#google-sign-in-btn');
```

Comment:
extreme nit: maybe stick with one of login, log-in or sign-in?
e.g. `... logInButton = ... sign-in ...` to `... loginButton = ... login ...`
If you do change this, taking off or adding a dash for `login` line 52 and 57 might help too.

Reply:
Agreed! Good catch